frontend/src/components/taskSelection/map.js

```diff
@@ -377,16 +400,13 @@ export const TasksMap = ({
         map.on('click', 'point-tasks-centroid-inner', () => navigate('./tasks'));
       }
 
-      map.on('mouseenter', 'tasks-fill', function (e) {
-        if (selectTask) {
-          // Change the cursor style as a UI indicator.
-          map.getCanvas().style.cursor = 'pointer';
-        }
-      });
       map.on('click', 'tasks-fill', onSelectTaskClick);
       map.on('mouseleave', 'tasks-fill', function (e) {
+        popup?.remove();
         // Change the cursor style as a UI indicator.
         map.getCanvas().style.cursor = '';
```

Comment:
[NEED HELP] Cursors are not being cleared when a popup is triggered - the cursor changes to a pointer instead of grab when the mouse hovers outside the project boundary, which is unanticipated. I've created a [JS fiddle](https://jsfiddle.net/jqhydzb6/5/) if you'd like to take a look.